assets/translations/footer.js

```diff
@@ -29,4 +29,10 @@ export default {
 		press: 'Press Kit',
 		charity: 'A 501(c)(3) Public Charity',
 	},
+	jp: {
+		credits: 'Credits',
+		contact: 'Contact Us',
+		press: 'Press Kit',
+		charity: 'A 501(c)(3) Public Charity',
+	},
```

Comment:
Leaving as English because I think English page header & footer are natural for Japanese website.